exporter/jaeger/example_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaeger_test
+
+import (
+	"log"
+
+	"go.opencensus.io/exporter/jaeger"
+	"go.opencensus.io/trace"
+)
+
+func Example() {
+	// Register the Jaeger exporter to be able to retrieve
+	// the collected spans.
+	exporter, err := jaeger.NewExporter(jaeger.Options{
+		Endpoint:    "http://localhost:14268",
+		ServiceName: "trace-demo",
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+	trace.RegisterExporter(exporter)
```

Comment:
defer trace.UnregisterExporter(exporter)

Reply:
We don't expect users to unregister often, hence I am leaving unregistering out.

Reply:
Hmm, it's nonetheless a good pattern to enforce since many people will be doing copy and paste from examples.

Reply:
What's the consequence of non unregistering? What is the use case for unregistering an exporter? I can't think of a reason anyone would want to do it and it complicates the implementation. I think we should consider removing the functionality wholly.

Reply:
I agree with this. Dynamically adding new backends and removing them might be a use case but I don't think it is a critical or common one.

Reply:
Dynamically adding exporters for a time period, rare case, for sure, but nonetheless the ability to undo a registration should still be left in. For exporters that we add and perhaps they use a lot of resources and need to be shut off, I think leaving trace.UnregisterExporter should still stay. An example is during root cause analyis or heavy traffic, your web app has an endpoint to trigger exporting for a period of time and when done, shut it off -- 2 years ago we had such a case with bugsnag in which there was a buggy exponential backoff implementation, that was triggered by the same heavy videos being transcoded. Instead of redeploying the entire app after instrumenting it, there was an endpoint to trigger exporting and then shut it off.